### ISSUE TITLE
Feature Request: Show Free Space on main window of every GUI client

### ISSUE THREAD

**Issue by Pentaphon** [transmission/transmission#5594]
### What is the issue?

It's about time that Transmission shows the current free space on the main window of every official GUI client. 

The WebUI, GTK app, Qt App, and MacOS app (if not already?) should always show the free space on the main windows instead of only showing it in the preferences menu. When torrenting, it's always useful to see how much space you have left on your destination folder and keeping that small but useful statistic on the main window at all times just makes sense on every client. 

Only 3rd party clients seem to do this, such as TransGUI. The official clients should also show free space remaining on the main window. 

### Which application of Transmission?

None

### Which version of Transmission?

N/A

**Comment by killemov** [comment]
Some time ago transmission acquired the 'Free space per location' buff, only without specifying 'location'. The only locations that ARE specified are the incomplete/complete download dirs. And yes, the free space for those should be presented more prominently.

**Comment by Pentaphon** [comment]
> And yes, the free space for those should be presented more prominently.

Yes, I think it should be coupled next to the speed indicators in all clients. There's plenty of room on all clients to add it there.

**Comment by killemov** [comment]
> Yes, I think it should be coupled next to the speed indicators in all clients. There's plenty of room on all clients to add it there.

Just like I did in [Shift](https://github.com/killemov/Shift). But how to show free space for all "locations" in a convenient or rather sensible way? I recently added some major path/location functionality that can scrape locations from all present torrents and show free space for them.

**Comment by Pentaphon** [comment]
> Just like I did in [Shift](https://github.com/killemov/Shift). But how to show free space for all "locations" in a convenient or rather sensible way? I recently added some major path/location functionality that can scrape locations from all present torrents and show free space for them.

Maybe if there's multiple locations, the free space indicator turns into a clickable dropdown to list all of the locations?
